Cubical/Data/Sigma/Properties.agda

```diff
@@ -47,107 +49,23 @@ mapˡ f (a , b) = (f a , b)
 Σ-split-iso : {x y : Σ A B}
```

Comment:
Shouldn't this just be \Sigma-Iso or something to match the naming below?

Reply:
I changed, but actually I think the names of `Σ≃` and `Σ≡` should be changed... they're not very informative.

Reply:
Any suggestions?

Reply:
Maybe ΣPath-iso, ΣPath≃, ΣPath≡, something like that?

Reply:
or PathΣ-etc

Reply:
Sounds good to me!

Reply:
Or maybe `ΣPathIso`?

Reply:
I guess to match the transport versions it should be `ΣPath≃PathΣ` and so on.

Reply:
Yeah, that sounds good!